Scripts/build-phases/generate-credentials.sh

```diff
@@ -16,10 +16,14 @@ PLIST_TEMPLATE_PATH=${SOURCE_ROOT}/Credentials/Templates/InfoPlist-Template.h
 BASH_INPUT_PATH=${SOURCE_ROOT}/Credentials/bash_secrets.tpl
 BASH_OUTPUT_PATH=${DERIVED_PATH}/bash_secrets
 
+SECRETS_PATH="${HOME}/.configure/woocommerce-ios/secrets/woo_app_credentials.json"
```

Comment:
`SECRETS_PATH` used to be a user defined build setting but used only in this script.

By moving the definition here, we save `n - 1` lines in the diff, where `n` is the number of build configurations. You can see that below in the project file (they are six, because the setting was defined in two targets and the project has three build configurations).

More importantly, by moving _stuff_ away from the project file, any diff related to that _stuff_ will be easier to read.